packages/lib-react-components/src/ZooFooter/ZooFooter.js

```diff
@@ -28,21 +29,26 @@ export const StyledEasterEgg = styled(Image)`
   }
 `
 
-export default function ZooFooter (props) {
-  const {
-    aboutNavListURLs,
-    adminContainer,
-    buildNavListURLs,
-    className,
-    getInvolvedNavListURLs,
-    newsNavListURLs,
-    policyNavListURLs,
-    projectNavListURLs,
-    talkNavListURLs,
-  } = props
-
+export default function ZooFooter ({
+  aboutNavListURLs,
+  adminContainer,
+  buildNavListURLs,
+  className,
+  getInvolvedNavListURLs,
+  locale,
+  newsNavListURLs,
+  policyNavListURLs,
+  projectNavListURLs,
+  talkNavListURLs
+}) {
   const { t } = useTranslation()
 
+  useEffect(() => {
+    if (locale) {
+      i18n.changeLanguage(locale)
```

Comment:
Will this still work when components are used in projects that aren’t Next apps? `lib-react-components` is published on npm so that it can be used independently from the Next apps.

Reply:
Yes, locale is simply a string in ZooFooter and does not rely on Next.js's router.